### ISSUE TITLE
ID as number or string field

### ISSUE THREAD

**Issue by jessica-azgs** [StraboSpot/strabo-server#2]
@jasonash You have the id as a number but we have it as a string in the mobile client. To get a pseudo-random id we're doing `key = new Date().getTime().toString();`

Is there supposed to be more structure to the id? Does what we're doing work for you as well? Or would it be better if we changed it a number field?

We'll have to sync up on the fields more than that of course but just trying to get some basics working now.

**Comment by jasonash** [comment]
OK, I have changed the API to use a string ID rather than integer.

One thing you might consider is to append a random integer to the end of 
your datetime string:

Date().getTime().toString()+Math.floor((Math.random() * 900) + 
100).toString()

I used datetime as a pseudo random identifier on a previous project and 
had an issue with duplicate IDs being assigned. Of course, at a 
millisecond level this shouldn't be much of an issue, but it is 
possible. Just a thought.

Jason
